core/src/main/java/org/apache/iceberg/rest/responses/LoadTableResponse.java

```diff
@@ -27,9 +27,6 @@
 import org.apache.iceberg.relocated.com.google.common.collect.Maps;
 import org.apache.iceberg.rest.RESTResponse;
 
-/**
- *
- */
```

Comment:
I think the correct fix is to add javadoc, not remove the placeholder.

Reply:
added javadoc